forte/data/entry_type_generator.py

```diff
@@ -39,3 +52,39 @@ def get_type_attributes():
 
         """
         return _get_type_attributes()
+
+    @staticmethod
+    def get_entry_attributes_by_class(input_entry_class_name: str):
+        """Get type attributes by class name. input_entry_class_name should be
+        an object class dotted name path.
+        Please note must add "data class" decorator to all the class and their
```

Comment:
Let's use the real name `dataclass` (remeber to use backtip), and provide a link to python: https://docs.python.org/3/library/dataclasses.html.  `dataclass` is a python concept and we need to follow the exact name. If you want to mention the decorator, literally just say `@dataclass`

Please also rephrase the sentence to make it grammatically correct, for example:

Note: This function is only applicable to classes decorated as Python `dataclass` since it relies on the `__dataclass_fields__` to find out the attributes.